tests/sles4sap/trento/cluster_deploy.pm

```diff
@@ -28,6 +28,11 @@ sub run {
         '-a', get_vnet($cluster_rg));
     record_info('NET PEERING');
     assert_script_run($cmd, 360);
+
+    my $prov = get_required_var('PUBLIC_CLOUD_PROVIDER');
+    my $inventory = qesap_get_inventory($prov);
+
+    qesap_ansible_cmd(cmd => 'crm status', provider => $prov, filter => $_), for ('vmhana01', 'vmhana02');
```

Comment:
There's an extra `,` there just before the `for`

Reply:
is it also **crm status** supposed to have the same output an all node?

Reply:
Yes, the output should be the same in all cluster nodes.